.github/workflows/tests.yml

```diff
@@ -26,6 +26,17 @@ jobs:
       - uses: psf/black@stable
       - uses: pre-commit/action@v3.0.0
 
+  bandit:
+    name: Bandit
+    needs: style
+    runs-on: ubuntu-latest
+    steps:
+      - uses: davidslusser/actions_python_bandit@v1.0.0
+        with:
+          src: "mne"
+          options: "-c pyproject.toml -ll -r"
```

Comment:
these options are
- `-c path/to/config/file`
- `-l` reports LOW and above, `-ll` MEDIUM and above, `-lll` HIGH and above
- `-r` recursive (all files under `src` directory)